File: kangaroo_app_sdk_web/lib/features/plugin_handler_list.dart
import 'package:kangaroo_app_sdk_web/base/plugin_channel_handler.dart';
import 'package:kangaroo_app_sdk_web/features/user_authentication/user_authentication_handler.dart';
import 'archive_inbox_item/archive_inbox_item_handler.dart';
import 'claim_offer/claim_offer_handler.dart';
import 'user_registration/user_registration_handler.dart';
import 'user_registration_with_crm/user_registration_with_crm_handler.dart';
import 'user_delete/user_delete_handler.dart';
import 'deposit_gift_card/gift_card_deposit_handler.dart';
import 'user_ala_carte/ala_carte_products_handler.dart';
import 'banners/banners_handler.dart';
import 'business/business_handler.dart';
import 'branches/branches_handler.dart';
import 'business_list/businesses_handler.dart';
import 'user_campaign_messages/campaign_messages_handler.dart';
import 'user_conglomerate_notification_preferences/get_conglomerate_notification_preferences_handler.dart';
import 'countries/countries_handler.dart';
import 'get_crm_fields/get_crm_fields_handler.dart';
import 'get_default_business_id/get_default_business_id_handler.dart';
import 'draws/draws_handler.dart';
import 'frequent_buyer_programs/frequent_buyer_programs_handler.dart';
import 'gift_cards/gift_cards_handler.dart';
import 'user_notification_preferences/get_notification_preferences_handler.dart';
import 'public_a_la_carte_products/public_ala_carte_products_handler.dart';
import 'public_banners/public_banners_handler.dart';
import 'public_branches/public_branches_handler.dart';
import 'public_business_settings/public_business_settings_handler.dart';
import 'public_offer/public_offer_handler.dart';
import 'public_offers/public_offers_handler.dart';
import 'public_products/public_products_handler.dart';
import 'public_reward/public_reward_handler.dart';
import 'public_rewards/public_rewards_handler.dart';
import 'social_media_links/social_media_links_handler.dart';
import 'get_spin_to_win_prizes/get_spin_to_win_prizes_handler.dart';
import 'strings/strings_handler.dart';
import 'tiers/tiers_handler.dart';
import 'user_transfer_messages/transfer_messages_handler.dart';
import 'user_business_ala_carte_products/user_business_ala_carte_products_handler.dart';
import 'user_business_banners/user_business_banners_handler.dart';
import 'user_business_frequent_buyer_programs/user_business_frequent_buyer_programs_handler.dart';
import 'user_business_gift_cards/user_business_gift_cards_handler.dart';
import 'user_business_offers/user_business_offers_handler.dart';
import 'user_business_products/user_business_products_handler.dart';
import 'user_business_profile/user_business_profile_handler.dart';
import 'user_business_rewards/user_business_rewards_handler.dart';
import 'user_business_social_media_links/user_business_social_media_links_handler.dart';
import 'user_business_tiers/user_business_tiers_handler.dart';
import 'user_coupons/user_coupons_handler.dart';
import 'user_details/user_details_handler.dart';
import 'user_offers/user_offers_handler.dart';
import 'user_products/user_products_handler.dart';
import 'user_profile/user_profile_handler.dart';
import 'user_rewards/user_rewards_handler.dart';
import 'user_transaction_history/user_transaction_history_handler.dart';
import 'user_used_coupons/user_used_coupons_handler.dart';
import 'make_pay_pal_payment/pay_pal_payment_handler.dart';
import 'perform_social_media_action/social_media_action_perform_handler.dart';
import 'post_offer_facebook_share/offer_facebook_share_handler.dart';
import 'make_gift_card_purchase/gift_card_purchase_handler.dart';
import 'user_transfer_recall/transfer_recall_handler.dart';
import 'redeem_coupon/redeem_coupons_handler.dart';
import 'redeem_reward/redeem_rewards_handler.dart';
import 'push_token_registration/push_token_registration_handler.dart';
import 'user_pin_reset_request/user_pin_reset_request_handler.dart';
import 'user_pin_reset_with_verification_code/user_pin_reset_handler.dart';
import 'user_scan_receipt/user_scan_receipt_handler.dart';
import 'user_balance_transfer/balance_transfer_handler.dart';
import 'user_unsubscribe/user_unsubscribe_handler.dart';
import 'update_default_business_id/update_default_business_id_handler.dart';
import 'user_notification_preferences_update/update_notification_preferences_handler.dart';
import 'user_pin_update/user_pin_update_handler.dart';
import 'user_profile_update/user_profile_update_handler.dart';
import 'user_check_in/user_check_in_handler.dart';
import 'user_verification/user_account_verification_handler.dart';


List<PluginChannelHandler> pluginHandlerList = [
  ArchiveInboxItemHandler(),

  ClaimOfferHandler(),

  UserRegistrationHandler(),

  UserRegistrationWithCrmHandler(),

  UserDeleteHandler(),

  GiftCardDepositHandler(),

  AlaCarteProductsHandler(),

  BannersHandler(),

  BusinessHandler(),

  BranchesHandler(),

  BusinessesHandler(),

  CampaignMessagesHandler(),

  GetConglomerateNotificationPreferencesHandler(),

  CountriesHandler(),

  GetCrmFieldsHandler(),

  GetDefaultBusinessIdHandler(),

  DrawsHandler(),

  FrequentBuyerProgramsHandler(),

  GiftCardsHandler(),

  GetNotificationPreferencesHandler(),

  PublicAlaCarteProductsHandler(),

  PublicBannersHandler(),

  PublicBranchesHandler(),

  PublicBusinessSettingsHandler(),

  PublicOfferHandler(),

  PublicOffersHandler(),

  PublicProductsHandler(),

  PublicRewardHandler(),

  PublicRewardsHandler(),

  SocialMediaLinksHandler(),

  GetSpinToWinPrizesHandler(),

  StringsHandler(),

  TiersHandler(),

  TransferMessagesHandler(),

  UserBusinessAlaCarteProductsHandler(),

  UserBusinessBannersHandler(),

  UserBusinessFrequentBuyerProgramsHandler(),

  UserBusinessGiftCardsHandler(),

  UserBusinessOffersHandler(),

  UserBusinessProductsHandler(),

  UserBusinessProfileHandler(),

  UserBusinessRewardsHandler(),

  UserBusinessSocialMediaLinksHandler(),

  UserBusinessTiersHandler(),

  UserCouponsHandler(),

  UserDetailsHandler(),

  UserOffersHandler(),

  UserProductsHandler(),

  UserProfileHandler(),

  UserRewardsHandler(),

  UserTransactionHistoryHandler(),

  UserUsedCouponsHandler(),

  PayPalPaymentHandler(),

  SocialMediaActionPerformHandler(),

  OfferFacebookShareHandler(),

  GiftCardPurchaseHandler(),

  TransferRecallHandler(),

  RedeemCouponsHandler(),

  RedeemRewardsHandler(),

  PushTokenRegistrationHandler(),

  UserPinResetRequestHandler(),

  UserPinResetHandler(),

  UserScanReceiptHandler(),

  BalanceTransferHandler(),

  UserUnsubscribeHandler(),

  UpdateDefaultBusinessIdHandler(),

  UpdateNotificationPreferencesHandler(),

  UserPinUpdateHandler(),

  UserProfileUpdateHandler(),

  UserCheckInHandler(),

  UserAccountVerificationHandler(),

  UserAuthenticationHandler()
];

File: kangaroo_app_sdk_android/android/src/main/kotlin/com/kangaroo/flutterplugin/android/features/PluginHandlerList.kt
package com.kangaroo.flutterplugin.android.features

import com.kangaroo.flutterplugin.android.base.PluginChannelHandler
import com.kangaroo.flutterplugin.android.features.userAuthentication.UserAuthenticationHandler
import com.kangaroo.flutterplugin.android.features.claimOffer.ClaimOfferHandler
import com.kangaroo.flutterplugin.android.features.userRegistration.UserRegistrationHandler
import com.kangaroo.flutterplugin.android.features.userDelete.UserDeleteHandler
import com.kangaroo.flutterplugin.android.features.depositGiftCard.GiftCardDepositHandler
import com.kangaroo.flutterplugin.android.features.userAlaCarte.AlaCarteProductsHandler
import com.kangaroo.flutterplugin.android.features.banners.BannersHandler
import com.kangaroo.flutterplugin.android.features.business.BusinessHandler
import com.kangaroo.flutterplugin.android.features.branches.BranchesHandler
import com.kangaroo.flutterplugin.android.features.businessList.BusinessesHandler
import com.kangaroo.flutterplugin.android.features.userCampaignMessages.CampaignMessagesHandler
import com.kangaroo.flutterplugin.android.features.userConglomerateNotificationPreferences.GetConglomerateNotificationPreferencesHandler
import com.kangaroo.flutterplugin.android.features.countries.CountriesHandler
import com.kangaroo.flutterplugin.android.features.getDefaultBusinessId.GetDefaultBusinessIdHandler
import com.kangaroo.flutterplugin.android.features.frequentBuyerPrograms.FrequentBuyerProgramsHandler
import com.kangaroo.flutterplugin.android.features.giftCards.GiftCardsHandler
import com.kangaroo.flutterplugin.android.features.userNotificationPreferences.GetNotificationPreferencesHandler
import com.kangaroo.flutterplugin.android.features.publicALaCarteProducts.PublicAlaCarteProductsHandler
import com.kangaroo.flutterplugin.android.features.publicBanners.PublicBannersHandler
import com.kangaroo.flutterplugin.android.features.publicBranches.PublicBranchesHandler
import com.kangaroo.flutterplugin.android.features.publicBusinessSettings.PublicBusinessSettingsHandler
import com.kangaroo.flutterplugin.android.features.publicOffers.PublicOffersHandler
import com.kangaroo.flutterplugin.android.features.publicProducts.PublicProductsHandler
import com.kangaroo.flutterplugin.android.features.publicRewards.PublicRewardsHandler
import com.kangaroo.flutterplugin.android.features.socialMediaLinks.SocialMediaLinksHandler
import com.kangaroo.flutterplugin.android.features.getSpinToWinPrizes.GetSpinToWinPrizesHandler
import com.kangaroo.flutterplugin.android.features.strings.StringsHandler
import com.kangaroo.flutterplugin.android.features.tiers.TiersHandler
import com.kangaroo.flutterplugin.android.features.userTransferMessages.TransferMessagesHandler
import com.kangaroo.flutterplugin.android.features.userBusinessAlaCarteProducts.UserBusinessAlaCarteProductsHandler
import com.kangaroo.flutterplugin.android.features.userBusinessBanners.UserBusinessBannersHandler
import com.kangaroo.flutterplugin.android.features.userBusinessFrequentBuyerPrograms.UserBusinessFrequentBuyerProgramsHandler
import com.kangaroo.flutterplugin.android.features.userBusinessGiftCards.UserBusinessGiftCardsHandler
import com.kangaroo.flutterplugin.android.features.userBusinessOffers.UserBusinessOffersHandler
import com.kangaroo.flutterplugin.android.features.userBusinessProducts.UserBusinessProductsHandler
import com.kangaroo.flutterplugin.android.features.userBusinessProfile.UserBusinessProfileHandler
import com.kangaroo.flutterplugin.android.features.userBusinessRewards.UserBusinessRewardsHandler
import com.kangaroo.flutterplugin.android.features.userBusinessSocialMediaLinks.UserBusinessSocialMediaLinksHandler
import com.kangaroo.flutterplugin.android.features.userCoupons.UserCouponsHandler
import com.kangaroo.flutterplugin.android.features.userDetails.UserDetailsHandler
import com.kangaroo.flutterplugin.android.features.userOffers.UserOffersHandler
import com.kangaroo.flutterplugin.android.features.userProducts.UserProductsHandler
import com.kangaroo.flutterplugin.android.features.userProfile.UserProfileHandler
import com.kangaroo.flutterplugin.android.features.userRewards.UserRewardsHandler
import com.kangaroo.flutterplugin.android.features.userTransactionHistory.UserTransactionHistoryHandler
import com.kangaroo.flutterplugin.android.features.userUsedCoupons.UserUsedCouponsHandler
import com.kangaroo.flutterplugin.android.features.makePayPalPayment.PayPalPaymentHandler
import com.kangaroo.flutterplugin.android.features.performSocialMediaAction.SocialMediaActionPerformHandler
import com.kangaroo.flutterplugin.android.features.postOfferFacebookShare.OfferFacebookShareHandler
import com.kangaroo.flutterplugin.android.features.makeGiftCardPurchase.GiftCardPurchaseHandler
import com.kangaroo.flutterplugin.android.features.userTransferRecall.TransferRecallHandler
import com.kangaroo.flutterplugin.android.features.redeemCoupon.RedeemCouponsHandler
import com.kangaroo.flutterplugin.android.features.redeemReward.RedeemRewardsHandler
import com.kangaroo.flutterplugin.android.features.pushTokenRegistration.PushTokenRegistrationHandler
import com.kangaroo.flutterplugin.android.features.userPinResetRequest.UserPinResetRequestHandler
import com.kangaroo.flutterplugin.android.features.userPinResetWithVerificationCode.UserPinResetHandler
import com.kangaroo.flutterplugin.android.features.userScanReceipt.UserScanReceiptHandler
import com.kangaroo.flutterplugin.android.features.userBalanceTransfer.BalanceTransferHandler
import com.kangaroo.flutterplugin.android.features.userUnsubscribe.UserUnsubscribeHandler
import com.kangaroo.flutterplugin.android.features.updateDefaultBusinessId.UpdateDefaultBusinessIdHandler
import com.kangaroo.flutterplugin.android.features.userNotificationPreferencesUpdate.UpdateNotificationPreferencesHandler
import com.kangaroo.flutterplugin.android.features.userPinUpdate.UserPinUpdateHandler
import com.kangaroo.flutterplugin.android.features.userProfileUpdate.UserProfileUpdateHandler
import com.kangaroo.flutterplugin.android.features.userCheckIn.UserCheckInHandler
import com.kangaroo.flutterplugin.android.features.userVerification.UserAccountVerificationHandler

val pluginHandlerList: List<PluginChannelHandler> = listOf(
    ClaimOfferHandler(),

    UserRegistrationHandler(),

    UserDeleteHandler(),

    GiftCardDepositHandler(),

    AlaCarteProductsHandler(),

    BannersHandler(),

    BusinessHandler(),

    BranchesHandler(),

    BusinessesHandler(),

    CampaignMessagesHandler(),

    GetConglomerateNotificationPreferencesHandler(),

    CountriesHandler(),

    GetDefaultBusinessIdHandler(),

    FrequentBuyerProgramsHandler(),

    GiftCardsHandler(),

    GetNotificationPreferencesHandler(),

    PublicAlaCarteProductsHandler(),

    PublicBannersHandler(),

    PublicBranchesHandler(),

    PublicBusinessSettingsHandler(),

    PublicOffersHandler(),

    PublicProductsHandler(),

    PublicRewardsHandler(),

    SocialMediaLinksHandler(),

    GetSpinToWinPrizesHandler(),

    StringsHandler(),

    TiersHandler(),

    TransferMessagesHandler(),

    UserBusinessAlaCarteProductsHandler(),

    UserBusinessBannersHandler(),

    UserBusinessFrequentBuyerProgramsHandler(),

    UserBusinessGiftCardsHandler(),

    UserBusinessOffersHandler(),

    UserBusinessProductsHandler(),

    UserBusinessProfileHandler(),

    UserBusinessRewardsHandler(),

    UserBusinessSocialMediaLinksHandler(),

    UserCouponsHandler(),

    UserDetailsHandler(),

    UserOffersHandler(),

    UserProductsHandler(),

    UserProfileHandler(),

    UserRewardsHandler(),

    UserTransactionHistoryHandler(),

    UserUsedCouponsHandler(),

    PayPalPaymentHandler(),

    SocialMediaActionPerformHandler(),

    OfferFacebookShareHandler(),

    GiftCardPurchaseHandler(),

    TransferRecallHandler(),

    RedeemCouponsHandler(),

    RedeemRewardsHandler(),

    PushTokenRegistrationHandler(),

    UserPinResetRequestHandler(),

    UserPinResetHandler(),

    UserScanReceiptHandler(),

    BalanceTransferHandler(),

    UserUnsubscribeHandler(),

    UpdateDefaultBusinessIdHandler(),

    UpdateNotificationPreferencesHandler(),

    UserPinUpdateHandler(),

    UserProfileUpdateHandler(),

    UserCheckInHandler(),

    UserAccountVerificationHandler(),

    UserAuthenticationHandler()
)
